### ISSUE TITLE
Adding urlencoded querystring parameter with Request Transformer plugin

### ISSUE THREAD

**Issue by vbaranov** [Mashape/kong#821]
I use Kong latest version 0.5.x.
I prepared simple API that uses only *Request Transformer* and *File Log* plugins.
I try to add urlencoded querystring parameter with this plugin.

For example:
`--data "config.add.querystring=where:%7B%22foo%22%3A%22bar%22%7D"`

In log file from *File Log* plugin I see that request query string is empty: `"querystring":{}`

Could you please explain what am I doing wrong? How to pass urlencoded querystring parameter through Request Transformer plugin?

Thanks in advance

**Comment by thibaultCha** [comment]
I was not able to reproduce under Kong 0.5.4.

API:
```
{
    "created_at": 1452741342000,
    "id": "e5e66dd3-774b-46aa-c6a8-294f0413ae4c",
    "name": "mockbin",
    "request_path": "/mockbin",
    "strip_request_path": true,
    "upstream_url": "http://mockbin.com"
}
```

Plugins:
```
{
    "data": [
        {
            "api_id": "e5e66dd3-774b-46aa-c6a8-294f0413ae4c",
            "config": {
                "add": {
                    "querystring": [
                        "hello:world",
                        "where:%7B%22foo%22%3A%22bar%22%7D"
                    ]
                }
            },
            "created_at": 1452741409000,
            "enabled": true,
            "id": "3d37be6d-0994-46df-c612-383964e12fcd",
            "name": "request-transformer"
        },
        {
            "api_id": "e5e66dd3-774b-46aa-c6a8-294f0413ae4c",
            "config": {
                "path": "/path/to/kong/log.log"
            },
            "created_at": 1452741461000,
            "enabled": true,
            "id": "f518d475-b0a5-41bd-c1c1-e9dc9321dd64",
            "name": "file-log"
        }
    ]
}
```

In the request going to upstream and in the log file, I clearly see:
```
"querystring":{"hello":"world","where":"%7B%22foo%22%3A%22bar%22%7D"}
``

**Comment by vbaranov** [comment]
Steps to reproduce at kong 0.5.4:
1. `curl -i -X POST --url http://localhost:8001/apis/ --data 'name=test' --data 'upstream_url=http://helloworld.com/' --data 'request_host=test'`
2. `curl -X POST http://localhost:8001/apis/test/plugins --data "name=request-transformer" --data "config.add.querystring=where:%7B%22foo%22%3A%22bar%22%7D"`
3. Postman api plugins [request](https://monosnap.com/file/pncO5dumfZFQl2zTw40FCO3qX4eQC3) (as you can see ":" is not escaped inside parameter value )
4. And as from the Request Transformer Plugin source code:
```
local parts = stringy.split(entry, ":")
for _, entry in ipairs(val) do
      local parts = stringy.split(entry, ":")
      cb(parts[1], utils.table_size(parts) == 2 and parts[2] or nil)
end
```
Not escaped ":" inside parameter brakes request because of keys/values split by ":".

But as I see, it is fixed in kong 0.6.0
